content/kit/wiring.md

```diff
@@ -1,3 +1,14 @@
 ---
 title: Wiring
 ---
+
+Ensuring your robot is wired correctly is incredibly important. If wired incorrectly, your kit may not work correctly, or may be damaged.
+
+## Wire Colouring
+Ground should be Black, and nothing else should be black. This is in the interest of both safety and organisation. Being able to see where all your ground cables allows you to easily track cable runs and diagnose electronic issues. It also greatly reduces the chance of running an actual current down the ground, which will serious damage the kit!
+
+## Keep your solders clean
+Sometimes, you'll need to build your own circuits to extend out kit, which likely requires soldering. Keep an eye on your solders so they don't create a bridge between components. This will likely lead to the electronics not working correctly.
+
+## Use higher gauge wire
+If a component is likely to draw a lot of current, it's best to give it a thicker wire. The motor board is likely to draw the most current, and so should use a very thick wire (like what is in your kit). Using a wire that's too thin will lead to your motor board trying to draw too much current and stop working. Using thinner wire will be fine for small electronics, such as micro-switches. 
```

Comment:
Using a thin wire could lead to overheating -> Fire hazard?? (@kierdavis I'm not too sure how hot these things could get.. is this correct?)

Reply:
If they tried to power the motor board over the thin solid-core wire, then yes there is. Although a part of me thinks that's a little closer to natural selection to worry about ;)